climada/entity/exposures/test/test_base.py

```diff
@@ -477,6 +484,12 @@ def test_set_crs(self):
         self.assertRaises(ValueError, probe.set_crs, 'epsg:3395')
         self.assertTrue(u_coord.equal_crs('EPSG:4326', probe.meta.get('crs')))
 
+    def test_to_crs_epsg_crs(self):
+        """ Check that if crs and epsg are both provided a ValueError is raised"""
+        with self.assertRaises(ValueError) as cm:
+            Exposures.to_crs(self,crs='GCS', epsg=26915, inplace=False)
+        self.assertEqual("one of crs or epsg must be None",
+                      str(cm.exception))
```

Comment:
```suggestion
        self.assertEqual("one of crs or epsg must be None", str(cm.exception))
```
We suppose the editor can hold 100 characters (that's also the project's pylint configuration). If the line break is meant for readability I'd rather break before `"one of crs..."`

Reply:
👍 Sorry for being nitty-gritty at the moment. That's just for the beginning, while still hoping we converge :grin:

Reply:
Thank you Emanuel for the suggestions. Don't worry, it's better to build good habits from the start